### PR TITLE
CR-8211

### PR DESCRIPTION
**Pull request by elad-codefresh**

### CODE REVIEW

File: docs/releases/release_notes.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
--   Argo CD [v2.1.3](https://github.com/codefresh-io/argo-cd/releases/tag/v2.1.3)
-    -   Argo CD ApplicationSet Controller [v0.2.0](https://github.com/argoproj-labs/applicationset/releases/tag/v0.2.0)
--   Argo Events [v1.4.0](https://github.com/argoproj/argo-events/releases/tag/v1.4.0)
--   Argo Rollouts [v1.1.0](https://github.com/argoproj/argo-rollouts/releases/tag/v1.1.0)
--   Argo Workflows [v3.1.8](https://github.com/argoproj/argo-workflows/releases/tag/v3.1.8)
+- Argo CD [v2.1.3](https://github.com/codefresh-io/argo-cd/releases/tag/v2.1.3)
+  - Argo CD ApplicationSet Controller [v0.2.0](https://github.com/argoproj-labs/applicationset/releases/tag/v0.2.0)
+- Argo Events [v1.4.0](https://github.com/argoproj/argo-events/releases/tag/v1.4.0)
+- Argo Rollouts [v1.1.0](https://github.com/argoproj/argo-rollouts/releases/tag/v1.1.0)
+- Argo Workflows [v3.1.8](https://github.com/argoproj/argo-workflows/releases/tag/v3.1.8)
```
Comment (ghost) [inline docs/releases/release_notes.md]: please change the following versions - 
argocd - v2.1.15. 
argo-events - v1.5.5 (https://github.com/codefresh-io/argo-events/releases/tag/v1.5.5-cap-CR-7756). 
argo-workflows - v3.2.4 (https://github.com/codefresh-io/argo-workflows/releases/tag/v3.2.4-cap-CR-6798)

**Review by noam-codefresh — APPROVED** [review]

**Pull request merged**